pyproject.toml

```diff
@@ -51,6 +51,19 @@ pylint = "*"
 pytest = "*"
 pytest-cov = "*"
 requests-mock = "^1.9"
+aiohttp = "^3.8.4"
+aiofiles = "^23.1.0"
+aiocsv = "^1.2.4"
+loguru = "^0.7.0"
+aiolimiter = "^1.1.0"
+aiobotocore = "^2.5.2"
```

Comment:
I think that these dependencies should be in the `tool.poetry.dependencies` section.
I suggest setting these dependencies as optional and use the [`extra`](https://python-poetry.org/docs/pyproject/#extras) to allow customers to install these dependencies only when needed